### ISSUE TITLE
Database refresh?

### ISSUE THREAD

**Issue by seltmann** [scan-bugs-org/North-American-Arthropod-Biodiversity#4]
Should we get a fresh version of the SCAN database in January before starting with the next stages of analyses?

**Comment by neilcobb** [comment]
@seltmann 

Yep, Evin can do it whenever we are ready.  Have to update a couple of snaphot collections next week, but Evin has to do some coding to filter records better.

**Comment by neilcobb** [comment]
@seltmann @lindsiebug 

I forgot to ask "How do we initiate a Data Carpentry workshop?"  or more specifically can you initiate one?
